main/check/APKBUILD

```diff
@@ -2,30 +2,25 @@
 # Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
 pkgname=check
 pkgver=0.12.0
-pkgrel=1
+pkgrel=2
 pkgdesc="A unit test framework for C"
 url="https://libcheck.github.io/check/"
 arch="all"
 license="LGPL-2.0-or-later"
 makedepends="pkgconfig"
 checkdepends="diffutils gawk"
-subpackages="$pkgname-doc $pkgname-dev"
+subpackages="$pkgname-static $pkgname-doc $pkgname-dev"
```

Comment:
I wasn't around for the entire `-static` discussion but does it really make sense to move a `76.0K` ar archive to a separate subpackage?

Reply:
Yes, the point of removing it from -dev is to avoid accidentally linking to it.

But in the case of check there isn't much point since check is compiled statically into tests most of the time